packages/doctor/lib/env.js

```diff
@@ -15,9 +15,18 @@ class EnvVarAndPathCheck extends DoctorCheck {
     if (typeof varValue === 'undefined') {
       return nok(`${this.varName} environment variable is NOT set!`);
     }
-    return (await fs.exists(varValue))
-      ? ok(`${this.varName} is set to: ${varValue}`)
-      : nok(`${this.varName} is set to '${varValue}' but this is NOT a valid path!`);
+
+    if (await fs.exists(varValue)) {
+      return ok(`${this.varName} is set to: ${varValue}`);
+    } else {
+      let err_msg = `${this.varName} is set to '${varValue}' but this is NOT a valid path!`;
+      // On Windows, when the env ver has %LOCALAPPDATA$, fs.exists cannot resolve the path.
```

Comment:
couple typos

```suggestion
      // On Windows, when the env var has %LOCALAPPDATA%, fs.exists cannot resolve the path.
```

but... why not?